pkg/nsx/services/vpc/store.go

```diff
@@ -30,18 +30,36 @@ func indexFunc(obj interface{}) ([]string, error) {
 	res := make([]string, 0, 5)
 	switch o := obj.(type) {
 	case *model.Vpc:
-		return filterTag(o.Tags), nil
+		return filterTagBy(o.Tags, common.TagScopeNamespaceUID), nil
 	case *model.LBService:
-		return filterTag(o.Tags), nil
+		return filterTagBy(o.Tags, common.TagScopeNamespaceUID), nil
 	default:
 		return res, errors.New("indexFunc doesn't support unknown type")
 	}
 }
 
-var filterTag = func(v []model.Tag) []string {
+func vpcIndexNamespaceNameFunc(obj interface{}) ([]string, error) {
+	switch o := obj.(type) {
+	case *model.Vpc:
+		return filterTagBy(o.Tags, common.TagScopeNamespace), nil
+	default:
+		return nil, errors.New("VPCIndexNamespaceFunc doesn't support unknown type")
+	}
+}
+
+func vpcIndexNamespaceIDFunc(obj interface{}) ([]string, error) {
```

Comment:
`vpcIndexNamespaceIDFunc` is the same with `indexFunc`, but we don't see usage for indexFunc`. It's better to check with @seanpang-vmware  and @TaoZou1 
There is no reason to keep duplicated funcs here.

Reply:
This function `indexFunc` is only used in test files and is unnecessary. I deleted it.